src/dotVariant.Generator/templates/IEnumerable.scriban-cs

```diff
@@ -15,6 +15,10 @@ end
 namespace {{ Options.ExtensionClassNamespace }}
 {
 {{~ end ~}}
+    /// <summary>
+    /// Extensions which allow for easy and powerful integration into `System.Linq`-like queries
+    /// on `IEnumerable&lt;T&gt;` sequences, that let you manipulate a stream of variants based on the contained type.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Extensions which allow for easy and powerful integration into `System.Linq`-like queries
    /// on <see cref="global::System.Collections.Generic.IEnumerable{T}" /> sequences, that let you manipulate a stream of variants based on the contained type.
    /// </summary>
```

Reply:
actually makes sense to use "see" here too